dataprofiler/data_readers/json_data.py

```diff
@@ -276,36 +277,35 @@ def _load_data_from_file(self, input_file_path: str) -> List:
                 )
             return data
 
-    def _get_data_as_records(self, data: List) -> List[str]:
+    def _get_data_as_records(self, data: Union[pd.DataFrame, Dict, List]) -> List[str]:
         """
         Extract the data as a record format.
 
         :param data: raw data
         :type data: list
         :return: dataframe in record format
         """
-        _data: Union[pd.DataFrame, List]
-        _data = self._get_data_as_df(data)
-        _data = _data.to_dict(orient="records", into=OrderedDict)
-        for i, sample in enumerate(_data):
-            _data[i] = json.dumps(
+        data = self._get_data_as_df(data)
```

Comment:
no longer having two variables of the data with `_data`
fixes type acceptance at the top